resources/definitions/zonestar_z10s_printer.def.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "ZONESTAR Z10S",
```

Comment:
I can't find any information about this printer. Can you provide a link to a spec sheet or purchasing page somewhere so that we can check this definition file?